common/lib/capa/capa/capa_problem.py

```diff
@@ -789,7 +793,9 @@ def _extract_html(self, problemtree):  # private
             answervariable = None
             if problemid in self.correct_map:
                 pid = input_id
-                status = self.correct_map.get_correctness(pid)
+                if not self.saved_answers:
```

Comment:
Note: This was tested around correctness.  This will not impact the correctness state in the correct map and will respect the current score.  It will return an unanswered state when an answer has been saved since last submit.